### PR TITLE
typos and lint errors

### PR DESCRIPTION
**Pull request by sfc-gh-chu**
# Description

fixes some typos/lint errors

## Other details good to know for developers

Please include any other details of this change useful for _TruLens_ developers.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
  not work as expected)
- [ ] New Tests
- [ ] This change includes re-generated golden test results
- [ ] This change requires a documentation update

### CODE REVIEW

File: src/core/trulens/core/instruments.py [hunk 1]
```diff
@@ -969,15 +969,17 @@ def instrument_bound_methods(self, obj: object, query: Lens):
                             )
                             if inspect.iscoroutinefunction(func):
 
-                                async def bound(*args, **kwargs):
+                                async def abound(*args, **kwargs):
```
Comment (sfc-gh-pmardziel) [inline src/core/trulens/core/instruments.py:972]: Thats an oopsie.

File: src/core/trulens/instrument/custom.py [hunk 2]
```diff
@@ -0,0 +1,543 @@
+"""Custom class application
```
Comment (sfc-gh-pmardziel) [inline src/core/trulens/instrument/custom.py:1]: Did this change get pulled in from another PR?

File: src/core/trulens/core/schema/record.py [hunk 3]
```diff
@@ -163,7 +163,7 @@ class Record(serial.SerialModel, Hashable):
     def __init__(
         self, record_id: Optional[mod_types_schema.RecordID] = None, **kwargs
     ):
-        super().__init__(record_id="temporary", **kwargs)
+        super(serial.SerialModel).__init__(record_id="temporary", **kwargs)
```
Comment (sfc-gh-pmardziel) [inline src/core/trulens/core/schema/record.py:166]: Was this a linter suggestion because this class inherits from more than one parent?

**Review by sfc-gh-pmardziel — APPROVED** [review]
Left a few comments.

**Review by sfc-gh-jreini — APPROVED** [review]

File: src/core/trulens/core/schema/record.py (hunk 3 above)
Comment (sfc-gh-chu) [reply src/core/trulens/core/schema/record.py:166]: yea

File: src/core/trulens/instrument/custom.py (hunk 2 above)
Comment (sfc-gh-chu) [reply src/core/trulens/instrument/custom.py:1]: oh yea looks like it
